epilepsy12/models/initial_assessment.py

```diff
@@ -18,65 +19,86 @@ class InitialAssessment(TimeStampAbstractBaseClass, UserStampAbstractBaseClass):
 
     """
 
-    date_of_initial_assessment = models.DateField(
-        "On what date did the initial assessment occur?",
-        null=True,
-        default=None
-    )
-    general_paediatrics_referral_made = models.BooleanField(
-        "date of referral to general paediatrics",
-        null=True,
-        default=None
-    )
-    date_of_referral_to_general_paediatrics = models.DateField(
-        "date of referral to general paediatrics",
-        null=True,
-        default=None
-    )
+    # date_of_initial_assessment = models.DateField(
```

Comment:
@eatyourpeas are these commented-out sections for future use or deprecated?

Reply:
leaving these for the moment but likely to remove in a future commit. sorry if they confuse.